Lib/subprocess.py

```diff
@@ -758,7 +760,8 @@ def __init__(self, args, bufsize=-1, executable=None,
                  startupinfo=None, creationflags=0,
                  restore_signals=True, start_new_session=False,
                  pass_fds=(), *, user=None, group=None, extra_groups=None,
-                 encoding=None, errors=None, text=None, umask=-1, pipesize=-1):
+                 encoding=None, errors=None, text=None, umask=-1, pipesize=-1,
+                 force_hide=False):
```

Comment:
An exception should be raised if `force_hide` is true in POSIX. For example, in the POSIX parameter checking that starts on line 791:
```python
if force_hide:
    raise ValueError("force_hide is only supported on Windows "
                     "platforms")
```

Reply:
Nice catch

Reply:
Remember to add the test in `POSIXProcessTestCase.test_invalid_args()`.

Reply:
Thanks. Going to mark as draft again until I get access to a Windows machine to test.